deploy/Chart/values.yaml

```diff
@@ -71,3 +71,14 @@ rp:
     deleteRetryDelaySeconds: 60
   terraform:
     path: "/terraform"
+
+dashboard:
+  enabled: true
```

Comment:
Do we want this to be enabled by default all the time, even when installed by Helm? The dashboard container adds CPU/memory usage to clusters and when running in a production environment operators may not want to add the container and the security footprint that it also represents.

Would it make sense to have `rad init` enable the dashboard (usually used creating a dev environment), but keep it disabled by default when you do `helm upgrade` (usually used when deploying a production Radius)

Reply:
deferring to @willtsai

Reply:
I think there's some value in keeping it consistent with an opt-out model so that users don't have to figure out when it's opt-in and when it would be opt-out. thoughts? @AaronCrawfis 

hypothetical question: would it be possible to have an opt-in mechanism for `helm upgrade`? @willdavsmith

Reply:
> I think there's some value in keeping it consistent with an opt-out model so that users don't have to figure out when it's opt-in and when it would be opt-out.

I think there are different users though. An operator wanting a production-ready environment may not want the dashboard, at least at this stage of the project. We've done good work on limiting the size of the dashboard image in terms of disc-footprint, CPU, and memory, but it's still a large image. A developer building a new application will definitely want the dashboard to learn and experiment on. But in the end it's definitely an optional component.

For comparison Dapr made a similar call when they removed the dashboard from their chart: https://github.com/dapr/dapr/issues/4724

Reply:
interesting! I didn't know dapr made this change. how about this - we'll keep the initial work to have dashboard "opt-out" for this release (so folks can try it out), and I will propose the 2 options (dashboard installs with Radius, dashboard installs with `rad dashboard init`) in a design review in the next 2 weeks. Whatever we go forward with, 0.32 will have full support for `rad dashboard` commands. does this sound alright? @willtsai @AaronCrawfis

Reply:
@willdavsmith this sounds like a fine idea to me, any objections @AaronCrawfis ?

Reply:
Sure, sounds good to me

Reply:
> I will propose the 2 options (dashboard installs with Radius, dashboard installs with rad dashboard init) in a design review in the next 2 weeks

Can I make a suggestion? What if we filed an issue and let it sit in the backlog until someone asks for it? This doesn't feel like it's absolutely required.